Update codes with o3

ricardoak25 opened the pull request:
**What current issue(s) from Github does this address?**

**What problem does this PR solve?**

**How did you solve this problem?**

**How did you make sure your solution works?**

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

- [ ] Unit tests written?

saltyskip commented:
Can you explain a bit more the goal of this PR? It is just cleaning up the codes?

ricardoak25 commented:
- Updated NEO-Swift project with https://github.com/O3Labs/OzoneWalletIOS
- Removed old codes
- Refacted O3Labs codes to avoid using O3Labs API's (It not the latest version for that reason)
- Refacted Unit-Test to accept new codes
- Refacted classes name
- Updated swift 3 codes to swift 4 codes
- Added GetToken and GetBalance (old neo-swift codes)
- Removed claim and sendAsset methods
